knexfile.js

```diff
@@ -0,0 +1,47 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/play-app',
+    migrations: {
+      directory: './db/migrations'
+    },
+    useNullAsDefault: true
+    }
+  };
+
+  staging: {
```

Comment:
I'm not sure if we will need all of this but lets leave it in for now until we know for sure.